production/helm/loki/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Entries should be ordered as follows:
 
 Entries should include a reference to the pull request that introduced the change.
 
+## 3.0.1
+
+- [BUGFIX] Fix Prometheus rule `LokiTooManyCompactorsRunning` aggregating multiple services together.
```

Comment:
```suggestion
- [BUGFIX] Fix Prometheus rule `LokiTooManyCompactorsRunning` aggregating multiple services together in Loki helm chart.
```

Just to be more clear about, this is not changing the alerts for other environments like `tanka` and only for helm charts where we have `loki-headless`.